hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -109,9 +109,9 @@
       sw_test: sw/device/tests/aes_test
     }
     {
-      name: chip_consecutive_irqs_test
+      name: chip_dif_plic_sanitytest
       uvm_test_seq: chip_sw_base_vseq
-      sw_test: sw/device/tests/consecutive_irqs_test
+      sw_test: sw/device/tests/dif/dif_plic_sanitytest
```

Comment:
This probably does not work. The `dif` directory should not be added IMO.

Reply:
@sriyerg Can you talk through the problem a bit more? I'm keen to understand the barriers to following this structure and approach.

This structure was presented as part of the [checklist](https://docs.opentitan.org/doc/project/checklist/#diftestsanity), and you seemed to agree at that point, including to the approach of starting with a hw-written test and then moving it to this proposed structure.

Reply:
Should be:
```suggestion
      sw_test: sw/device/tests/dif_plic_sanitytest
```

The elf target is one level up.